CIP-interoperable-programmable-tokens/README.md

```diff
@@ -0,0 +1,172 @@
+---
+CIP: ???
+Title: Interoperable Programmable Tokens
+Category: Plutus
+Status: Proposed
+Authors:
+    - Philip DiSarro <philipdisarro@gmail.com>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/444
+Created: 2024-12-3
+License: Apache-2.0
+---
+
+<!-- Existing categories:
+
+- Meta     | For meta-CIPs which typically serves another category or group of categories.
+- Wallets  | For standardisation across wallets (hardware, full-node or light).
+- Tokens   | About tokens (fungible or non-fungible) and minting policies in general.
+- Metadata | For proposals around metadata (on-chain or off-chain).
+- Tools    | A broad category for ecosystem tools not falling into any other category.
+- Plutus   | Changes or additions to Plutus
+- Ledger   | For proposals regarding the Cardano ledger (including Reward Sharing Schemes)
+- Catalyst | For proposals affecting Project Catalyst / the Jörmungandr project
+
+-->
+
+## Abstract
+
+This CIP proposes a robust framework for the issuance of interoperable programmable tokens on Cardano. Unlike all its predecessors, this framework allows these tokens to be used in existing dApps, and does not require dApps to be developed specifically for these tokens. 
+
+## Motivation: why is this CIP necessary?
+
+This CIP proposes a solution to Cardano Problem Statement 3 ([CPS-0003](https://github.com/cardano-foundation/CIPs/pull/382/files?short_path=5a0dba0#diff-5a0dba075d998658d72169818d839f4c63cf105e4d6c3fe81e46b20d5fd3dc8f)).
```

Comment:
@colll78 I've resuscitated candidate CPS-0003 here, and as you recall from the CIP meeting today we'll expedite its review & merge, so it will be OK to include all the CPS-0003 references:
- https://github.com/cardano-foundation/CIPs/pull/947